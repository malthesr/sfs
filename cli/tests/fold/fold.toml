args = [
  "fold",
  "--precision",
  "1",
  "tests/fold/two_populations.sfs"
]
