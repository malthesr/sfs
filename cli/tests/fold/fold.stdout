#SHAPE=<5/5>
0.5 0.8 0.6 1.1 0.6 0.8 0.6 1.4 0.6 NaN 0.4 0.6 0.2 NaN NaN 0.8 0.6 NaN NaN NaN 0.6 NaN NaN NaN NaN
